src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -130,4 +132,21 @@ public static Tag parseTag(String tag) throws ParseException {
     public static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
         return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
     }
+
+    /**
+     * Parses a {@code String indexes} into a {@code List<Index>}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code address} is invalid.
+     */
+    public static List<Index> parseIndexes(String indexes) throws ParseException {
+        requireNonNull(indexes);
+        String[] indexStringArray = indexes.trim().split(" ");
+        List<Index> indexArray = new ArrayList<>();
```

Comment:
Can be shortened to 
```java
return Arrays.stream(arr)
  .map(ParserUtil::parseIndex)
  .collect(Collectors.toList());
```

Reply:
Tried using stream but parseIndex throws an exception. Apparently unhandled exception does poorly with streams unless I explicitly throw it inside the lambda.